app/scripts/modules/amazon/serverGroup/configure/wizard/securityGroups/securityGroupsRemoved.directive.js

```diff
@@ -12,12 +12,18 @@ module.exports = angular
       scope: {},
       bindToController: {
         command: '=',
+        removed: '=',
       },
       controllerAs: 'vm',
       controller: 'awsServerGroupSecurityGroupsRemovedCtrl',
     };
   }).controller('awsServerGroupSecurityGroupsRemovedCtrl', function () {
     this.acknowledgeSecurityGroupRemoval = () => {
-      this.command.viewState.dirty.securityGroups = null;
+      if (this.command && this.command.viewState && this.command.viewState.dirty) {
```

Comment:
good point. I was trying to avoid bringing in lodash, but it's a lot more readable that three `&&`s